templates/deploy-ansible-service-broker.template.yaml

```diff
@@ -317,8 +317,11 @@ objects:
         - type: local_openshift
           name: localregistry
           namespaces: ['openshift']
+          # NOTE TO ADMINS: The default whitelist policy here will pass *all* APBs
+          # found in the local openshift registry. If this is not desired,
+          # manipulate the following regex to only match APBs you wish to be made available.
           white_list:
-            - ".*-apb$"
```

Comment:
Does it work if you just comment it out? Maybe just add it to the block comment.

Reply:
It would if you removed the `white_list` altogether, or just commented it out, but I sort of feel like it's not a bad thing to ask admins who would like to tweak this to write their own regex. Also there is an example immediately above for dockerhub.

Reply:
I agree with you.